metamodel-uitleg.md

```diff
@@ -220,6 +213,13 @@ En ook de rol(invulling) zelf kunnen we zien als een domeinobject. Zo is een eig
 > - De invulling van deze eigenschap voor deze betaling is [Bakkerij Broodjes]
 > - [Bakkerij Broodjes] is in deze betaling de betaler.
 
+Er zijn eigenschappen die gaan over een onderscheid tussen objecten die onveranderlijk is. Zo zal de eigenschap «geboortedatum» van een persoon nooit veranderen: een persoon is nu eenmaal geboren op een bepaalde dag, dat zal niet meer veranderen. De eigenschap «leeftijd» verandert daarentegen elk jaar, maar wel op een hele reguliere manier (elk jaar eentje erbij). Er zijn ook eigenschappen die veranderen, zonder dat sprake is van vooraf duidelijke manier. Zo zijn rollen over het algemeen veranderlijk: [Jan] is (nu) werknemer van [Bakkerij Broodjes], maar mogelijk is hij dat over een tijdje niet meer, en er is ook een tijd geweest dat hij het (nog) niet was.
+
+Zo zijn er ook categorische kenmerken die onveranderlijk zijn en categorische kenmerken die juist veranderlijk zijn. Zo zal een levend wezen altijd tot de categorie van zoogdieren behoren, of juist niet. De categorie «werkeloos» is echt juist weer veranderlijk, en hangt bovendien ook samen met de rol «werknemer».
```

Comment:
Voor deze alinea geldt hetzelfde. Neem hem op in de paragraaf met de titel 'Classificaties en categoriën' of eveneens in een nieuwe paragraaf hier.